azure-pipelines.yml

```diff
@@ -9,14 +9,28 @@ trigger:
 jobs:
 - job: WinBuild
   displayName: Windows Build
-  pool:
-    vmImage: 'vs2017-win2016'
   strategy:
     matrix:
-      Debug:
-        CI_ENV_BUILD_TYPE: Debug
-      Release with Release Installer:
-        CI_ENV_BUILD_TYPE: Release
+#        windows-2022-Debug:
+#            imageName: "windows-2022"
+#            CI_ENV_BUILD_TYPE: Debug
+#        windows-2022-Release:
+#            imageName: "windows-2022"
+#            CI_ENV_BUILD_TYPE: Release
+#        windows-2019-Debug:
+#            imageName: "windows-2019"
+#            CI_ENV_BUILD_TYPE: Debug
+#        windows-2019-Release:
+#            imageName: "windows-2019"
+#            CI_ENV_BUILD_TYPE: Release
+        windows-2016-Debug:
```

Comment:
Let's limit ourselves only to release builds to save CI time.

Reply:
This has been resolved in the latest force-push (edited commit).